content/react/pt/get-started.md

```diff
@@ -44,34 +44,34 @@ yarn start
 ```
 
 <div class="aside">
-  Nota: Se <code>yarn test</code> emitir um erro, será necessário instalar o pacote <code>watchman</code>, tal como recomendado neste <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">problema</a>
+  Nota: Se o comando <code>yarn test</code> produzir um erro, será necessário instalar o pacote <code>watchman</code>, conforme recomendado nesta <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">thread</a>
```

Comment:
Why change the "problema" para "thread"? It's best to leave it as is. Both audiences can read it perfectly and understand the meaning. The rest of the phrase looks good.

Reply:
This link refers to yarn but is not a specific problem of it.
The main problem is about the npm and in the middle of the thread they talk about the yarn.

Reply:
> The main problem is about the npm and in the middle of the thread they talk about the yarn.
This phrase exists, due to the fact that the guide is heavily influenced by Create React App and with that as they are much more prone to using yarn as opposed to npm, for reasons that are out of the scope of this guide. The writter left in that note. That sentence is being corrected to allow it both ways. Using npm or yarn